presto-bigquery/pom.xml

```diff
@@ -358,6 +358,7 @@
                                 <exclude>com.fasterxml.jackson.core:jackson-core</exclude>
                                 <exclude>javax.annotation:javax.annotation-api</exclude>
                                 <exclude>com.fasterxml.jackson.core:jackson-databind</exclude>
+                                <exclude>org.apache.commons:commons-lang3</exclude>
```

Comment:
Can you explain why this is needed?  Can you try to remove it please?

Reply:
To maintain consistency across dependencies, I initially excluded commons-lang3 in the submodule, but it was already defined as a direct dependency in the pom.xml. The correct approach would have been to simply upgrade its version to ensure consistency across dependencies. I have now updated the version properly.